scm-ui/ui-api/src/help/search/syntax.en.ts

```diff
@@ -136,10 +136,10 @@ Search supports using parentheses to group clauses to form sub queries. This can
 
 The search supports escaping special characters that are part of the query syntax. The current list special characters are
 
-+ - && || ! ( ) { } [ ] ^ " ~ * ? : \\ /
+&plus; &minus; && || ! ( ) { } [ ] ^ " ~ * ? : &bsol;&bsol; /
 
-To escape these characters use the "\\" before the character. For example to search for (1+1):2 use the query:
+To escape these characters use the "&bsol;&bsol; " before the character. For example to search for (1+1):2 use the query:
```

Comment:
Remove space before `"`
```suggestion
To escape these characters use the "&bsol;" before the character. For example to search for (1+1):2 use the query:
```